dashboard/app/(dj)/setbuilder/components/curve.module.css

```
@@ -3,8 +3,92 @@
+  background: rgba(0, 245, 212, 0.7);
+  border-color: #00f5d4;
+}
+
+.srOnly {
+  position: absolute;
+  width: 1px;
+  height: 1px;
+  padding: 0;
+  margin: -1px;
+  overflow: hidden;
+  clip: rect(0, 0, 0, 0);
```

Stylelint (17.12.0)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)